public/css/list/redundancy-group-list-item.less

```diff
@@ -0,0 +1,20 @@
+.redundancy-group-list-item {
+  .caption {
+    display: flex;
+    justify-content: space-between;
+
+    .plugin-output {
+      .line-clamp(2);
```

Comment:
please add a comment, explaining that this is necessary due to the caption not being a -webkit-box anymore